aiohttp/client.py

```diff
@@ -939,6 +939,21 @@ async def _ws_connect(
             assert conn is not None
             conn_proto = conn.protocol
             assert conn_proto is not None
+
+            # For WS connection the sock_read must be either receive_timeout
+            # or timeout (whichever is specified), unless read_timeout is greater
+            # None == no timeout, i.e. infinite timeout
+            if ws_timeout.ws_receive is None:
+                # Reset regardless
+                conn_proto.read_timeout = None
+            elif conn_proto.read_timeout is None:
```

Comment:
Is a bit unexpected to write this with a pass. It would be cleaner to reverse the condition and remove this block

Reply:
I wrote this with a pass for the comment for logic clarity. I can of course reverse it if you'd like.

Reply:
Yes, please reverse